lib/file-helpers.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  sparqlEscapeUri,
+  uuid,
+  sparqlEscapeString,
+  sparqlEscapeDateTime,
+  sparqlEscapeInt,
+} from 'mu';
+import { join } from 'path';
+import { mkdir } from 'fs/promises';
+import { writeFile, stat } from 'fs/promises';
+import { querySudo as query, updateSudo as update } from '@lblod/mu-auth-sudo';
+import { MU_SPARQL_ENDPOINT, OPARL_TO_ELI_SERVICE_URI } from '../constants';
+const connectionOptions = {
+  sparqlEndpoint: MU_SPARQL_ENDPOINT,
+  mayRetry: true,
+};
+
+/**
+ * Write the given TTL content to a file and relates it to the given remote file and submitted document
+ *
+ * @param string graph Graph to write the content to
+ * @param string content Content to write to the file
+ * @param string logicalFileName Logical file name
+ * @param string sourceFile URI of the source file to relate the new file to
+ * @param string folderId Optional folder ID to organize files (defaults to empty string)
+ * @param string contentType Content type of the file (defaults to "text/turtle")
+ * @param string extension File extension (defaults to "ttl")
+ * @param string subfolder Optional subfolder within extract directory (defaults to empty string)
+ */
+export async function writeFileToTriplestore(
+  graph,
+  content,
+  logicalFileName,
+  sourceFile,
+  folderId = '',
+  contentType = 'text/turtle',
+  extension = 'ttl',
+  subfolder = '',
+) {
+  const baseFolder = subfolder
+    ? join('/share', folderId, 'extract', subfolder)
+    : join('/share', folderId, 'extract');
+  await mkdir(baseFolder, { recursive: true });
+
+  const phyId = uuid();
```

Comment:
phy for physical? i'd always recommend writing the names out in full, your IDE will help you with auto-completion but a reader's brain may not have that feature

Reply:
done (rewritten to physicalFileId and same for lo)